test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1504,6 +1525,11 @@ class test_ucp_am_nbx_rndv_dts : public test_ucp_am_nbx_rndv {
 
         test_ucp_am_nbx::cleanup();
     }
+
+    virtual unsigned enable_proto()
+    {
+        return 0;
```

Comment:
shall we test both new and old protocols for datatypes too?

Reply:
It hasn't been implemented yet. I'll enable tests for new protocol within adding support of iov and generic datatypes.

Reply:
I see, thanks